core/src/main/java/tachyon/Constants.java

```diff
@@ -64,4 +67,9 @@
    * Version 3 [0.6.0] Add lastModificationTimeMs to inode.
    */
   public static final int JOURNAL_VERSION = 2;
+
+  public static final Class<? extends DataServer> DEFAULT_DATA_SERVER_CLASS =
```

Comment:
There are several consts in this file relevant only to master/worker
I can create new files for UserConstants(will contain only 1 const for now), MasterConstants and WorkerConstants. though it sounds like an overkill to me
Or I can add them to the WorkerConf and UserConf but i'm not sure they belong there